modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -14,47 +14,49 @@ The API access requester button component enables members to request access to a
 
 Members can request access for an existing client application and API instance, or request to create a new client application by specifying the application name and description and choosing an API instance.
 
-* *API*: Choose the API.
-* *Button Border Radius*: Set the radius of the button border. A larger radius makes the button look more rounded.
-* *Button Align*: Set button alignment to left, right, or center.
-* *Show Modal Title Line*: Show the title line in the access request modal.
+API:: Choose the API.
+Button Border Radius:: Set the radius of the button border. A larger radius makes the button look more rounded.
+Button Align:: Set button alignment to left, right, or center.
+Show Modal Title Line:: Show the title line in the access request modal.
+
 // check description of Show Modal Title Line
```

Comment:
Weird about the italic in the transform. We'll have to ask Fer. Don't worry about it for now.